templates/ezid/journal_content.xml

```diff
@@ -16,7 +16,9 @@
             <journal_metadata>
                 <full_title>{{ article.journal.name }}</full_title>
                 <abbrev_title>{{ article.journal.name }}</abbrev_title>
+                {% if article.journal.issn %}
```

Comment:
Please check with program managers if we should also check for '0000-0000'. That is the default issn.

Reply:
@justingonder confirms that we also do not want to send over the default ISSN to Crossref. I'll add that to this PR.

Reply:
Thanks for confirming, Hardy!